### ISSUE TITLE
Is it possible add  weather layers to leaflet maps  using flutter.

### ISSUE THREAD

**Issue by Gaushikmr** [BenBitDesign/flutter_leaflet_examples#1]

**Comment by BenBitDesign** [comment]
@Gaushikmr Could you be more specific? If you mean adding icons showing the weather like ☁️, ☀️, 🌧 this can be achieved in the same way as the example where the flutter icon is on the map. But then simply replacing that with your own icon for the weather.

**Comment by Gaushikmr** [comment]
NOt icons, I need to add the weather layers. https://openweathermap.org/api/weathermaps
Is it possible to add open weather map layers to leaflet using flutter

**Comment by BenBitDesign** [comment]
Have you tried changing the openstreetmap URL? That could work for your use case.

For example here:
![image](https://user-images.githubusercontent.com/15856201/131265975-91477ec1-572b-41e5-847f-e879cf685c31.png)

Please note that I am by no means in charge of the support of the leaflet plugin for Flutter and I simply made an own public version of the examples.

**Comment by Gaushikmr** [comment]
Yes, I tried its showing the tiles but i wanted the yiles to be displahed
in maps.
